### PR TITLE
qa/Fixed upgrade sequence to 10.2.0 -> 10.2.7 -> latest -x (10.2.8)

### PR DESCRIPTION
**Pull request by yuriw**
Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

### CODE REVIEW

**Comment by yuriw** [comment]
testing http://pulpito.front.sepia.ceph.com:80/yuriw-2017-07-03_22:01:11-upgrade:jewel-x:point-to-point-x-jewel-distro-basic-vps/

File: qa/suites/upgrade/jewel-x/point-to-point-x/point-to-point-upgrade.yaml [hunk 1]
```diff
@@ -202,13 +205,7 @@ upgrade-sequence_x:
    - ceph.restart: [osd.4]
    - sleep:
        duration: 30
-   - ceph.restart:
-       daemons: [osd.5]
-       wait-for-healthy: false
-       wait-for-up-osds: true
-   - exec:
-       mon.a:
-         - ceph osd set require_jewel_osds
```
Comment (smithfarm) [inline qa/suites/upgrade/jewel-x/point-to-point-x/point-to-point-upgrade.yaml]: This deleted block needs to be added at the end of `upgrade-sequence_jewel` so it runs when the final daemon (`mon.c`) is upgraded to v10.2.7.

Comment (yuriw) [inline qa/suites/upgrade/jewel-x/point-to-point-x/point-to-point-upgrade.yaml]: done

**Comment by yuriw** [comment]
@smithfarm test passed

**Pull request merged**